.github/workflows/ci.yml

```diff
@@ -140,6 +140,7 @@ jobs:
     - name: Upload coverage report
       if: matrix.python-version == 3.5 && github.repository == 'aiidateam/aiida-core'
       uses: codecov/codecov-action@v1
+      continue-on-error: true  # don't fail job, if coverage upload fails
```

Comment:
Shouldn't we rather set line `147` to `false`?

Reply:
I think one refers to the upload process itself and the other one allows us to fail the CI if the coverage is insufficient.

Reply:
From [their documentation](https://github.com/codecov/codecov-action#arguments):
> `fail_ci_if_error`: Specify if CI pipeline should fail when Codecov runs into errors during upload. Defaults to false

I interpret this to mean that us setting this to `True` is exactly what is failing the pipeline when the upload false. Or am I misreading this?

Reply:
@sphuber , thanks for pointing this out - indeed the action has already intrinsic support for this.
Will fix

Reply:
Should have look it up instead of just guessing, sorry about that.